app/models/measure_condition.rb

```diff
@@ -3,5 +3,10 @@
 class MeasureCondition
   include ApiEntity
 
-  attr_accessor :condition_code, :condition, :document_code, :requirement, :action, :duty_expression
+  attr_accessor :condition_code, :condition, :document_code, :action, :duty_expression
+  attr_writer :requirement
+
+  def requirement
+    @requirement.html_safe
```

Comment:
We've talked about the extensive use of html_safe in this code base and agreed that there is a risk to users of the service if any of our downstream import processes are compromised. We should do a review of where we use html_safe and make sure we're sanitising script tags.

Reply:
Yeah, there's always that risk.